### PR TITLE
release-schema.json: add hasNonDisclosureAgreement, nonDisclosureAgre…

### PR DESCRIPTION
**Pull request by odscjen**
…ement, CustomerService, and contractTerms.customerServices

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,18 @@
...
 
+If you are using the [Lots extension](https://extensions.open-contracting.org/en/extensions/lots/master/), [follow its guidance](https://extensions.open-contracting.org/en/extensions/lots/master/#usage) on whether to use `tender.lots` fields or `tender` fields.
+
 ## Guidance
 
 ## Legal context
 
-In the European Union, this extension's fields correspond to [eForms BG-711 (Contract Terms)](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/), although not all the fields have been implemented yet. See [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/master/en/) for the correspondences to Tenders Electronic Daily (TED).
+In the European Union, this extension's fields correspond to [eForms BG-711 (Contract Terms), BT-801 (Non Disclosure Agreement), BT-802 (Non Disclosure Agreement Description), OPT-071 (Quality Target Code) and OPT-072 (Quality Target Description)](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/). For correspondences to eForms fields, see [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/).
```
Comment (duncandewhurst) [inline README.md]: As in the other PRs, I think that we should preserve the reference to OCDS for the European Union.

File: release-schema.json [hunk 2]
```diff
@@ -143,9 +143,72 @@
             "null"
           ],
           "minLength": 1
+        },
+        "hasNonDisclosureAgreement": {
+          "title": "Has non disclosure agreement",
```
Comment (duncandewhurst) [inline release-schema.json]:
```suggestion
          "title": "Has non-disclosure agreement",
```

File: release-schema.json [hunk 3]
```diff
@@ -143,9 +143,72 @@
...
+          "description": "Whether or not the contract execution will require the use of a non disclosure agreement.",
```
Comment (duncandewhurst) [inline release-schema.json]:
```suggestion
          "description": "Whether contract execution requires a non-disclosure agreement.",
```

File: release-schema.json [hunk 4]
```diff
@@ -143,9 +143,72 @@
...
+          "type": [
+            "boolean",
+            "null"
+          ]
+        },
+        "nonDisclosureAgreement": {
+          "title": "Non disclosure agreement",
```
Comment (duncandewhurst) [inline release-schema.json]:
```suggestion
          "title": "Non-disclosure agreement",
```

File: release-schema.json [hunk 5]
```diff
@@ -143,9 +143,72 @@
...
+          "description": "The nature and extent of the non disclosure agreement required as part of the contract execution.",
```
Comment (duncandewhurst) [inline release-schema.json]:
```suggestion
          "description": "The nature and extent of the non-disclosure agreement required as part of the contract execution.",
```

Comment (duncandewhurst) [reply release-schema.json]: Is the 'nature and extent' wording taken from somewhere? I wonder if we should have a more generic description in line with the title and description of BT-802, e.g. "A description of the non-disclosure agreement required for contract execution".

File: release-schema.json [hunk 6]
```diff
@@ -143,9 +143,72 @@
...
+          "type": [
+            "string",
+            "null"
+          ],
+          "minLength": 1
+        },
+        "customerServices": {
+          "title": "Customer services",
+          "description": "The customer services the supplier is required to supply as part of this contract.",
```
Comment (duncandewhurst) [inline release-schema.json]:
```suggestion
          "description": "Information about customer service requirements.",
```

File: release-schema.json [hunk 7]
```diff
@@ -143,9 +143,72 @@
...
+          "items": {
+            "$ref": "#/definitions/CustomerService"
+          }
         }
       },
       "minProperties": 1
+    },
+    "CustomerService": {
+      "title": "Customer service",
+      "description": "Information about a customer service requirement linked to this contract.",
```
Comment (duncandewhurst) [inline release-schema.json]:
```suggestion
      "description": "A customer service requirement.",
```

File: release-schema.json [hunk 8]
```diff
@@ -143,9 +143,72 @@
...
+      "minProperties": 1,
+      "type": "object",
+      "properties": {
+        "type": {
+          "title": "Type",
+          "description": "The code that defines the type of customer service requirement.",
```
Comment (duncandewhurst) [inline release-schema.json]:
```suggestion
          "description": "The type of this customer service requirement.",
```

File: release-schema.json [hunk 9]
```diff
@@ -143,9 +143,72 @@
...
+          "type": [
+            "string",
+            "null"
+          ],
+          "minLength": 1
+        },
+        "name": {
+          "title": "Name",
+          "description": "The name of the customer service requirement.",
```
Comment (duncandewhurst) [inline release-schema.json]:
```suggestion
          "description": "A name for this customer service requirement.",
```

File: release-schema.json [hunk 10]
```diff
@@ -143,9 +143,72 @@
...
+          "type": [
+            "string",
+            "null"
+          ],
+          "minLength": 1
+        },
+        "description": {
+          "title": "Description",
+          "description": "The description of the customer service requirement.",
```
Comment (duncandewhurst) [inline release-schema.json]:
```suggestion
          "description": "A description of this customer service requirement.",
```

File: README.md [hunk 11]
```diff
@@ -23,18 +27,66 @@ In the European Union, this extension's fields correspond to [eForms BG-711 (Con
...
+
 ## Issues
 
 Report issues for this extension in the [ocds-extensions repository](https://github.com/open-contracting/ocds-extensions/issues), putting the extension's name in the issue's title.
 
 ## Changelog
 
+### 2023-03-10
+
+* Add `hasNonDisclosureAgreement`, `nonDisclosureAgreement`, `CustomerService` and `contractTerms.customerServices`.
```
Comment (duncandewhurst) [inline README.md]:
```suggestion
* Add `.hasNonDisclosureAgreement`, `.nonDisclosureAgreement` and `.customerServices` fields.
```

File: README.md [hunk 12]
```diff
@@ -23,18 +27,66 @@ In the European Union, this extension's fields correspond to [eForms BG-711 (Con
       "tendererLegalForm": "Contractors may jointly apply for the contract.",
       "hasExclusiveRights": false,
       "operatorRevenueShare": 0.25,
-      "socialStandards": "The supplier maintains the social, collective bargaining and labor law obligations according to Union law, national law or collective agreements. 4 paragraph 4a Regulation 13707/2007."
+      "socialStandards": "The supplier maintains the social, collective bargaining and labor law obligations according to Union law, national law or collective agreements. 4 paragraph 4a Regulation 13707/2007.",
+      "hasNonDisclosureAgreement": true,
+      "nonDisclosureAgreement": "A Non Disclosure Agreement will need to ...",
```
Comment (duncandewhurst) [inline README.md]:
```suggestion
      "nonDisclosureAgreement": "A non-disclosure agreement is required in order to...",
```

File: README.md [hunk 13]
```diff
@@ -23,18 +27,66 @@ In the European Union, this extension's fields correspond to [eForms BG-711 (Con
...
+          "electronicInvoicingPolicy": "required",
+          "reservedExecution": true,
+          "performanceTerms": "A set of KPIs will be developed for this contract and the successful tenderer will be measured against these for the duration of the contract. Please refer to briefing document for further details.",
+          "financialTerms": "In the event that a work referred to in § 2.6 of the Agreement is created as part of the implementation of the Subject Matter of the Agreement, the Contractor shall indicate on the invoice what proportion of the remuneration for implementation.",
+          "tendererLegalForm": "Contractors may jointly apply for the contract.",
+          "hasExclusiveRights": false,
+          "operatorRevenueShare": 0.25,
+          "socialStandards": "The supplier maintains the social, collective bargaining and labor law obligations according to Union law, national law or collective agreements. 4 paragraph 4a Regulation 13707/2007.",
+          "hasNonDisclosureAgreement": true,
+          "nonDisclosureAgreement": "A Non Disclosure Agreement will need to ...",
```
Comment (duncandewhurst) [inline README.md]:
```suggestion
          "nonDisclosureAgreement": "A non-disclosure agreement is required in order to...",
```

**Review by duncandewhurst — CHANGES_REQUESTED** [review]

File: release-schema.json (hunk 5 above)
Comment (odscjen) [reply release-schema.json]: It likely did come from somewhere but I can't find or remember where right now so I think you're rephrasing is better, I'll update that :)

File: README.md (hunk 12 above)
Comment (odscjen) [reply README.md]: This one I had lifted directly from the eForms example but happy to go with your suggestion

File: README.md (hunk 11 above)
Comment (odscjen) [reply README.md]: I did commit this but then the periods made the tests fail so I removed them again

Comment (duncandewhurst) [reply README.md]: Ah, sorry!

**Review by duncandewhurst — APPROVED** [review]

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**